src/main/java/firebase/sdk/dashboard/api/UserResource.java

```diff
@@ -57,8 +59,12 @@ public Response addUser(User user) {
   @Path("{uid}/sdks")
   public Response getUserFavorites(@PathParam("uid") String uid) {
     User user = USER_DAO.getUser(uid);
-    Map<String, List<String>> favourites = user.favoriteSDKs();
-    return ResponseHandler.createJsonResponse(Status.OK, favourites);
+    if (user != null) {
+      Map<String, List<String>> favourites = user.favoriteSDKs();
+      return ResponseHandler.createJsonResponse(Status.OK, favourites);
+    } else {
+     return ResponseHandler.createJsonResponse(Status.NO_CONTENT, Arrays.asList()); 
```

Comment:
I think you can use NOT_FOUND status here since the user resource is not found in this case.